api/src/main/java/org/openmrs/util/LoggingListener.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.util;
+
+import org.openmrs.GlobalProperty;
+import org.openmrs.api.GlobalPropertyListener;
+
+public class LoggingListener implements GlobalPropertyListener {
+
+	/**
+	 * Asks this listener whether it wants to be notified about the given property name
```

Comment:
How about using the `@see` annotation for the javadocs instead of duplicating them?